main/test/flix/Test.Exp.Unit.flix

```diff
@@ -1,13 +1,10 @@
 namespace Test/Exp/Unit {
 
     @test
-    def testUnit01(): () = ()
+    def testUnit01(): Unit = ()
```

Comment:
I think this is a good change, imo. I believe we had a ticket somewhere that discusses this.